src/applications/vaos/components/AddToCalendar.jsx

```diff
@@ -10,10 +11,16 @@ export default function AddToCalendar({
   startDateTime,
   duration,
 }) {
+  const instructions = PURPOSE_TEXT.some(purpose =>
```

Comment:
This will exclude the instructions on Community Care appointments, which we do want to pass through.